site/content/blog/2020-05-04-pritunl.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "From OpenVPN to Pritunl VPN: The transition"
+slug: pritunl
+date: 2020-05-04T12:00:00-04:00
+author: Angelos Kyratzakos
+github: angeloskyratzakos
+community: angelos.kyratzakos
+---
+
+Usually companies are using office's network to prevent unauthorized people to connect to their private network and by using their own network infrastructure/connectivity they can maintain their desirable level of security for their data. But it would be convenient to connect to "that private network" while they are away from office's building while using an internet connection. To solve that problem, VPN (Virtual Private Network)
```

Comment:
```suggestion
Usually organizations use an internal network to prevent unauthorized people from connecting to their private network and by using their own network infrastructure/connectivity they can maintain their desirable level of security for their data. But it would be convenient for users to connect to that private network while they are away from the office's building, on their own internet connection. To solve that problem, VPN (Virtual Private Network)
```